db/migrate/20131120225132_add_organization_fields.rb

```diff
@@ -1,6 +1,6 @@
 class AddOrganizationFields < ActiveRecord::Migration
   def change
-    add_column :taxonomies, :description, :text
+    add_column :taxonomies, :description, :text unless column_exists?(:taxonomies, :description)
```

Comment:
I don't think you need this change. I believe this should always run before `20141225073211_add_description_to_taxonomies.rb`